helpers/configureImage.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const URL = require('url').URL;
+const utils = require('handlebars-utils');
+const common = require('./lib/common.js');
+
+const factory = () => {
+    return function(image, options) {
+        if (!utils.isObject(image) || !utils.isString(image.data) || !common.isValidURL(image.data)) {
+            // Return empty string if this does not appear to be an image object.
+            return ''
+        }
+
+        // 'https://cdn0.bigcommerce.com/s-hash/images/stencil/{:size}/l/some-image.original.jpg?c=2'
+        let url = new URL(image.data);
+        
+        //remove AIM query strings if already set by the application
+        url.searchParams.delete('imbypass');
+        url.searchParams.delete('imdevicesize');
+        url.searchParams.delete('imdeviceformat');
+
+        if (options && utils.isOptions((options))) {
+            if (options.hash['disable-all-optimizations']) {
```

Comment:
Should we allow the theme developer set these values inside theme config (`globals.getThemeSettings()`) so it can apply to all cdn images ?

Reply:
I was imagining this helper would be used to override the behavior of particular images, not apply broadly across the theme. For example, people want to make certain images nicer for downloading so they might get rid of device formatting (`.webp` is not what you want to download).

Reply:
The reason I ask ... there are a lot stores which have had issues in the past with the way akamai image manager serves some of their images on various devices. Adding the setting at the global level gives them the opportunity to turn it off entirely if there are issue. But I just realized this is going to be a nested helper inside `getImage` so it won't even trigger to respect global settings unless we fetch the image inside `configureImage`.

Reply:
Can we make it more so that `disable-all-optimizations` and other properties are options on `getImage` so all of this happens behind the scenes.

Reply:
Yep I considered just making `{{getImage}}` more complicated to accommodate this. Perhaps we can do it once we have `{{getImageSrcset}}` and do it all at once.